test/dpg/version-tolerant/Expected/AcceptanceTests/DPGTestModelsVersionTolerant/dpgtestmodelsversiontolerant/aio/_operations/_operations.py

```diff
@@ -204,7 +205,7 @@ def get_pages(self, mode: str, **kwargs: Any) -> AsyncIterable["_models.Product"
         _headers = kwargs.pop("headers", {}) or {}
         _params = kwargs.pop("params", {}) or {}
 
-        cls = kwargs.pop("cls", None)  # type: ClsType[_models.ProductResult]
+        cls = kwargs.pop("cls", None)  # type: ClsType[ProductResult]
```

Comment:
Accroding to `1. Give the model a private name, so include _ at the front of the models name`, it shall be `# type: ClsType[_ProductResult]`

Reply:
I ended up changing it to not be a private model, I just don't expose it in `.models`, so users won't see it. It's a good point though, we can talk about it at scrum today. I'm going to go ahead and merge it like this for now, but will modify based off of what we end up deciding